src/main/java/org/openpnp/gui/tablemodel/PlacementsTableModel.java

```diff
@@ -111,8 +112,9 @@ else if (columnIndex == 4) {
                 placement.setLocation(location);
             }
             else if (columnIndex == 5) {
-                placement.setLocation(placement.getLocation().derive(null, null, null,
-                        Double.parseDouble(aValue.toString())));
+                /* placement.setLocation(placement.getLocation().derive(null, null, null,
```

Comment:
Is setting handled by the cell value class? If so, can this commented code be removed?